src/connectors/connector_aiplatform.workflows.yaml

```diff
@@ -12,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# [START workflows_connector_vertexai]
-# This workflow demonstrates how to use the Vertex AI connector.
+# [START workflows_connector_aiplatform]
```

Comment:
region tag renames, if workflows_connector_vertexai is used in docs currently, you need to either: 

1. freeze the snippet in docs so it doesn't error out when you rename the region tag
2. do the update in 2 phase. leave the existing region tag in place and then remove in a later update. 

In this case, you're also moving the file which means you probably need to freeze. If @camiekim is working with you and already doing this, awesome, just wanted to flag if this is not known. She can help you out with this so that docs don't break.

Reply:
This is a new connector and docs are not released to the public yet. Renaming the sample should be safe.

Reply:
Yes, you can go ahead and rename the region tag.